Code/Tools/ProjectManager/Platform/Windows/ProjectUtils_windows.cpp

```diff
@@ -92,12 +92,43 @@ namespace O3DE::ProjectManager
                 }
             }
 
-            return AZ::Failure(QObject::tr("Visual Studio 2019 version 16.9.2 or higher not found.\n\n"
+            return AZ::Failure(QObject::tr("Visual Studio 2019 version 16.9.2 or higher not found.<br><br>"
                 "Visual Studio 2019 is required to build this project."
                 " Install any edition of <a href='https://visualstudio.microsoft.com/downloads/'>Visual Studio 2019</a>"
                 " or update to a newer version before proceeding to the next step."
                 " While installing configure Visual Studio with these <a href='https://o3de.org/docs/welcome-guide/setup/requirements/#visual-studio-configuration'>workloads</a>."));
         }
         
+        AZ::Outcome<void, QString> OpenCMakeGUI(const QString& projectPath)
+        {
+            auto cmakeProcessEnvResult = GetCommandLineProcessEnvironment();
+            if (!cmakeProcessEnvResult.IsSuccess())
+            {
+                return AZ::Failure(cmakeProcessEnvResult.GetError());
+            }
+
+            QString projectBuildPath = QDir(projectPath).filePath(ProjectBuildPathPostfix);
+            AZ::Outcome result = GetProjectBuildPath(projectPath);
+            if (result.IsSuccess())
+            {
+                projectBuildPath = result.GetValue();
+            }
+
+            QProcess process;
+            process.setProcessEnvironment(cmakeProcessEnvResult.GetValue());
+
+            // if the project build path is relative, it should be relative to the project path 
+            process.setWorkingDirectory(projectPath);
+
+            process.setProgram("cmake-gui");
+            process.setArguments({ "-S", projectPath, "-B", projectBuildPath });
+            if(!process.startDetached())
+            {
+                return AZ::Failure(QObject::tr("Failed to start CMake GUI"));
+            }
+
+            return AZ::Success();
+        }
+
```

Comment:
The Linux implementation of this function could be the same.

Reply:
OK, I plan on finishing my setup today to test it on Linux, thanks!